public/js/modal.js

```diff
@@ -0,0 +1,20 @@
+// Get the modal
+var modal = document.getElementById("myModal");
```

Comment:
for javascript it's good to use single quote

Reply:
When you consistently use single quotes, you can avoid escaping the single quotes within your strings, which can make the code cleaner.

Reply:
Understood, sir.

Reply:
Thank you for the information, sir.